.github/workflows/main.yml

```diff
@@ -317,16 +317,22 @@ jobs:
       - name: Install Apt dependencies
         run: |
           sudo apt update
-          sudo apt install libboost-dev doxygen graphviz texlive-bibtex-extra
+          sudo apt install libboost-dev doxygen graphviz texlive-bibtex-extra \
+            libblas-dev liblapack-dev libhdf5-dev libfmt-dev libsundials-dev
       - name: Upgrade pip
         run: python3 -m pip install -U pip setuptools wheel
       - name: Install Python dependencies
         run: |
-          python3 -m pip install ruamel.yaml scons numpy cython sphinx==5.1.1 \
-          sphinxcontrib-matlabdomain sphinxcontrib-doxylink pint \
-          pydata-sphinx-theme==0.13.3 sphinx-argparse
-      - name: Build Cantera with documentation
-        run: python3 `which scons` build -j2 doxygen_docs=y sphinx_docs=y debug=n optimize=n use_pch=n
+          python3 -m pip install ruamel.yaml scons numpy cython sphinx==6.2.1 \
+            sphinxcontrib-matlabdomain sphinxcontrib-doxylink pint \
+            pydata-sphinx-theme==0.13.3 sphinx-argparse sphinx_design matplotlib \
+            pandas scipy
+          python3 -m pip install "git+https://github.com/speth/sphinx-gallery.git@multi-src-lang"
+          python3 -m pip install "git+https://github.com/speth/sphinx-tags.git@tags-with-spaces"
```

Comment:
Following up on the earlier conversation. I agree that it's definitely not necessary to wait for new upstream releases, but I'd suggest to wait until we can use official repos here.

Reply:
Per our discussion on Cantera/enhancements#178, these are now updated to use `sphinx-gallery/sphinx-gallery` and `Cantera/sphinx-tags`.